app/views/partials/_winner.html.haml

```diff
@@ -0,0 +1,3 @@
+.results
+  - data[:list].each do |team|
+    %p #{team[:team].name} got #{team[:score]} points
```

Comment:
I would love to see a reverse sort like we talked about - so the highest scores would be at the top.